wkcuber/api/Slice.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+from wkw import Dataset
+
+from wkcuber.api.TiffData.TiffMag import TiffMag
+
+
+class Slice:
+    def __init__(
+        self,
+        path_to_mag_dataset,
+        header,
+        size=(1024, 1024, 1024),
```

Comment:
Why does this default to 1024? What do you think about making this parameter non-optional?